forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas

```diff
@@ -0,0 +1,68 @@
+nagawarrior.png
+size: 64,96

```

Comment:
wrong size, causes crash with the adventure editor (which is a great tool to check sprite animations) 

64,80

Reply:
Thanks, fixing.